StoreApp.BLL/Services/ProductService.cs

```diff
@@ -58,6 +65,49 @@ public IEnumerable<ProductDTO> GetAll()
             return config.Map<IEnumerable<Product>, IEnumerable<ProductDTO>>(DataBase.Products.GetAll());
         }
 
+        public bool ValidateNewProduct(ProductDTO product)
+        {
+            var validate = DataBase.Products.Find(p => p.Name.Equals(product.Name) &&
+                                                       p.BrandId.Equals(product.BrandId) &&
+                                                       p.ProducerId.Equals(product.ProducerId));
```

Comment:
It is not the correct name of the variable cause you get a collection of the objects here. You can name it like **products**

Reply:
fixed